osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceRegistrationMessageService.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.core.application.services;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+
+import org.opensmartgridplatform.core.domain.model.domain.DomainRequestService;
+import org.opensmartgridplatform.domain.core.entities.Device;
+import org.opensmartgridplatform.domain.core.entities.DomainInfo;
+import org.opensmartgridplatform.domain.core.entities.Ssld;
+import org.opensmartgridplatform.domain.core.entities.SsldPendingFirmwareUpdate;
+import org.opensmartgridplatform.domain.core.repositories.DeviceRepository;
+import org.opensmartgridplatform.domain.core.repositories.DomainInfoRepository;
+import org.opensmartgridplatform.domain.core.repositories.SsldRepository;
+import org.opensmartgridplatform.shared.domain.services.CorrelationIdProviderTimestampService;
+import org.opensmartgridplatform.shared.infra.jms.RequestMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class DeviceRegistrationMessageService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeviceRegistrationMessageService.class);
+
+    private static final String LOCAL_HOST = "127.0.0.1";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1313)

Reply:
Won't fix; this IP address is only used to match IP addresses of devices. There is no need to make it configurable.